### DISCUSSION TITLE
Error RegistryKeyLookupFailed `CoderModulesPath'

### DISCUSSION THREAD

**Discussion by StylishRainbow** [ImageMagick/ImageMagick#6807] [Help]
### ImageMagick version

7.1.1-Q16

### Operating system, version and so on

Windows 11

### Description

I have reinstalled ImageMagick. It's the latest version. I've added the ImageMagick exe folder to my environment variables on my system as a precaution, but I get the error every time I try to use image magick on my system:

`magick: RegistryKeyLookupFailed `CoderModulesPath' @ error/module.c/GetMagickModulePath/667.
magick: no decode delegate for this image format `PNG' @ error/constitute.c/ReadImage/746.`

I've ran for the directories on my computer within command prompt to try and rectify the issue:

`set PATH=%PATH%;"C:\Program Files (x86)\ImageMagick-7.1.1-Q16"
set MAGICK_HOME="C:\Program Files (x86)\ImageMagick-7.1.1-Q16"
set MAGICK_CODER_MODULE_PATH="C:\Program Files (x86)\ImageMagick-7.1.1-Q16\modules\coders"
`

And this didn't help. If anyone could help I'd be greatly appreciative!

**Comment by dlemstra** [comment]
You should not be hitting that part when you set those folders. Did you double check that they exist and that your account has access to those files?

**Comment by krzcho** [comment]
There are windows registry entries needed for IM to run
I stumbled upon this error after copying IM folder to a new machine
